src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -389,7 +389,8 @@ namespace
         asr::Assembly&          assembly,
         const std::string&      instance_name,
         Mtl*                    mtl,
-        MaterialMap&            material_map)
+        MaterialMap&            material_map,
+        const bool              use_max_proc_maps)
```

Comment:
That's definitely a better name! As I mentioned in another comment, I'd go even further (and use the same name everywhere): `use_max_procedural_maps`